trin-core/src/portalnet/overlay_service.rs

```diff
@@ -365,7 +369,41 @@ impl OverlayService {
                     }
                 }
                 _ = OverlayService::bucket_maintenance_poll(self.protocol.clone(), &self.kbuckets) => {}
+                _ = bucket_refresh_interval.tick() => {
+                    debug!("Bucket refresh lookup");
```

Comment:
Probably won't want this in long-term, but I'm fine leaving it in while the code is fresh.